### PR TITLE
ET-268: Add identifier for requests from demo mobile sample app

### PR DESCRIPTION
**Pull request by michalpaukert**

### CODE REVIEW

File: Example/KenticoCloud/Utils/Extensions.swift [hunk 1]
```diff
@@ -54,3 +55,26 @@ extension Date {
...
+        return copyOfArray
+    }
+}
+
+
+class QueryStringParameter {
+    static func addSampleSite(param: String) -> String {
+        let lastChar = param.last!
+
+        if (lastChar == "/") {
```
Comment (makma) [inline Example/KenticoCloud/Utils/Extensions.swift]: what about:
`https://github.com/makma/cloud-sdk-swift/pull/1/commits/866901499497135f8fb957d4af22e292b2fe2ef7?asfaghg=1`

File: Example/KenticoCloud/ViewControllers/Article/ArticlesViewController.swift [hunk 2]
```diff
@@ -107,7 +107,7 @@ class ArticlesViewController: ListingBaseViewController, UITableViewDataSource {
     private func getArticles() {
         self.showLoader(message: "Loading articles...")
         
-        let customQuery = "items?system.type=article&order=elements.post_date[desc]"
+        let customQuery = QueryStringParameter.addSampleSite(param: "items?system.type=article&order=elements.post_date[desc]")
```
Comment (makma) [inline Example/KenticoCloud/ViewControllers/Article/ArticlesViewController.swift]: addSampleSiteQueryStringParameter

File: Example/KenticoCloud/Utils/Extensions.swift [hunk 3]
```diff
@@ -54,3 +55,26 @@ extension Date {
         return dateFormatter.string(from: self)
     }
 }
+
+extension Array where Iterator.Element == QueryParameter {
+    func addSampleSite() -> [QueryParameter] {
```
[the remark this answers is not in the document]
Comment (michalpaukert) [reply Example/KenticoCloud/Utils/Extensions.swift]: Fixed

File: Example/KenticoCloud/ViewControllers/Article/ArticlesViewController.swift (hunk 2 above)
Comment (michalpaukert) [reply Example/KenticoCloud/ViewControllers/Article/ArticlesViewController.swift]: fixed

File: Example/KenticoCloud/Utils/Extensions.swift (hunk 1 above)
Comment (michalpaukert) [reply Example/KenticoCloud/Utils/Extensions.swift]: I think it is not possible in easy way.
Delivery api does not allows you to add & after some query it only allows to add ?query=something

**Pull request merged**